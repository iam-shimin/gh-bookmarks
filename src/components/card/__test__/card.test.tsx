import React from 'react';
import { render } from '@testing-library/react';
import Card from '../card';

test('<Card />', () => {
	const { asFragment } = render(<Card />);
	expect(asFragment()).toMatchSnapshot();
});