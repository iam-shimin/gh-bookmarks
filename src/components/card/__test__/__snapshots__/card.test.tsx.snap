// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> 1`] = `
<DocumentFragment>
  <div
    class="card "
  />
</DocumentFragment>
`;
